scss/components/_sidebar.scss

```diff
@@ -86,7 +86,7 @@
   }
 
   .card {
-    @include u-font-color($ivory);
-    background-color: #DD3900;
+    @include u-font-color($cream);
+    background-color: #dd3700;
```

Comment:
Color literals like `#dd3700` should only be used in variable declarations; they should be referred to via variable everywhere else.